src/fonts.php

```diff
@@ -22,6 +22,72 @@ function str_ends_with( $haystack, $needle ) {
     }
 }
 
+if ( ! class_exists( 'Stackable_Theme_Fonts' ) ) {
+	class Stackable_Theme_Fonts {
+
+		public static $theme_fonts = [];
+
+		function __construct() {
+			if ( is_frontend() ) {
+				add_action( 'init', array( $this, 'gather_theme_fonts' ) );
+			}
+
+			add_action( 'rest_api_init', array( $this, 'register_rest_fields' ) );
+		}
+
+		public function gather_theme_fonts() {
+			if ( ! method_exists( 'WP_Font_Face_Resolver', 'get_fonts_from_theme_json' ) ) {
+				return false;
```

Comment:
No need to return anything in an action.
```suggestion
				return;
```